Library/Homebrew/diagnostic.rb

```diff
@@ -389,10 +389,10 @@ def check_homebrew_prefix
         return if HOMEBREW_PREFIX.to_s == "/usr/local"
 
         <<-EOS.undent
-          Your Homebrew is not installed to /usr/local
-          You can install Homebrew anywhere you want but some bottles (binary
-          packages) can only be used in /usr/local and some formulae (packages)
-          may not build correctly unless you install in /usr/local. Sorry!
+          Your Homebrew's prefix is not /usr/local.
```

Comment:
I'm fine with either but let's merge as-is and then iterate if people are confused (or happy for you to open a PR to modify 😁, just want to get this in for a tag tomorrow).